### PR TITLE
update(CSS): web/css/_colon_nth-child

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [:nth-child()@MDN](https://developer.mozilla.org/en-us/docs/Web/CSS/:nth-child), [сирці :nth-child()@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/css/_colon_nth-child/index.md)

Нові зміни:
- [mdn/content@764b291](https://github.com/mdn/content/commit/764b291098cfe575374580844f1a91e7d814a8d3)
- [mdn/content@8bc20cc](https://github.com/mdn/content/commit/8bc20cc2cd8ec3f1ffbac41d8f02dcceca06ae1b)
- [mdn/content@bb6092c](https://github.com/mdn/content/commit/bb6092c4230b69c2eceae6910af68c73955cae1c)
- [mdn/content@6990d43](https://github.com/mdn/content/commit/6990d43b6ef8f163bdddaf2382d54e27652a8724)
- [mdn/content@09afdce](https://github.com/mdn/content/commit/09afdcefc433569fd162f2bd54db2607f2256bec)

### CODE REVIEW

**Comment by undead404** [comment]
Розв'язано конфлікт

File: files/uk/web/css/_colon_nth-child/index.md [hunk 1]
```diff
@@ -134,30 +151,257 @@ browser-compat: css.selectors.nth-child
...
+
+#### Результат
+
+Елементи з `class="noted"` мають товсту нижню межу, а елементи 3, 10 і 17 мають суцільний фон, оскільки вони є _парними_ елементами списку з `class="noted"`.
+
+{{EmbedLiveSample('zastosuvannia-of-selector', 550, 120)}}
+
+### Запис of selector і запис selector nth-child
+
+In this example, there are two unordered lists of names. The first list shows the effect of `li:nth-child(-n + 3 of .noted)` and the second list shows the effect of `li.noted:nth-child(-n + 3)`.
```
Comment (Aenqau) [inline files/uk/web/css/_colon_nth-child/index.md:253]: Тут без перекладу поки?

**Review by Aenqau — APPROVED** [review]

File: files/uk/web/css/_colon_nth-child/index.md [hunk 2]
```diff
@@ -134,30 +151,257 @@ browser-compat: css.selectors.nth-child
...
+
+У цьому прикладі присутні два невпорядковані списки імен. Перший показує дію `li:nth-child(-n + 3 of .noted)`, а другий – дію `li.noted:nth-child(-n + 3)`.
```
Comment (undead404) [inline files/uk/web/css/_colon_nth-child/index.md:255]:
```suggestion
У цьому прикладі присутні два невпорядковані списки імен. Перший показує дію `li:nth-child(-n + 3 of .noted)`, а другий – дію `li.noted:nth-child(-n + 3)`.
```

File: files/uk/web/css/_colon_nth-child/index.md (hunk 1 above)
Comment (undead404) [reply files/uk/web/css/_colon_nth-child/index.md]: Виправив

**Pull request merged**